include/fiction/algorithms/simulation/sidb/assess_physical_popstability.hpp

```diff
@@ -0,0 +1,160 @@
+//
+// Created by Jan Drewniok on 02.11.23.
+//
+
+#ifndef FICTION_QUANTITY_STABILITY_OF_GROUND_STATE_HPP
+#define FICTION_QUANTITY_STABILITY_OF_GROUND_STATE_HPP
+
+#include "fiction/algorithms/simulation/sidb/quickexact.hpp"
+#include "fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp"
+#include "fiction/layouts/cell_level_layout.hpp"
+#include "fiction/traits.hpp"
+#include "fiction/types.hpp"
+
+#include <cmath>
+#include <limits>
+#include <tuple>
+#include <unordered_map>
+#include <utility>
+
+namespace fiction
+{
+
+/**
+ * This defines the various types of charge transitions that can occur in a SiDB layout. These transitions represent
+ * changes in the charge state of SiDBs, including transitions from neutral to negative, negative to neutral, neutral to
+ * positive, and positive to neutral.
+ */
+enum class transition_type
+{
+    NEUTRAL_TO_NEGATIVE,
+    NEGATIVE_TO_NEUTRAL,
+    NEUTRAL_TO_POSITIVE,
+    POSITIVE_TO_NEUTRAL,
+};
+
+/**
+ * This function calculates the minimum electrostatic potential to the closest charge transition for a SiDB layout at
+ * given physical parameters. It records information about the SiDB which is closest to a charge transition, the
+ * transition type, and the electrostatic potential needed for the charge transition.
```

Comment:
Based on this docstring, it is unclear to me what the function does, what it is good for, and what it returns. Why does it return a map? What do its entries mean?